tools/git_sha_rewriter.py

```diff
@@ -107,6 +108,9 @@ def RewriteBinary(path, offset, git5_sha1):
   if len(sys.argv) != 2:
     print 'Usage: %s <Envoy binary path> ' % sys.argv[0]
     sys.exit(1)
+  if platform.system() == 'Darwin':
+    print 'Stamping not supported on OSX'
```

Comment:
At a minimum, OS X binaries are not ELF format, so this particular tool will not work. I don't know enough about Mach-O to say whether it's possible to do something similar.